examples/async-requests/github.py

```diff
@@ -14,3 +14,19 @@ def get_repos(self):
     @uplink.get("/repos/{owner}/{repo}/contributors")
     def get_contributors(self, owner, repo):
         """Lists contributors for the specified repository."""
+
+# Executes the code above
+# The for loop is used for readability, you can just call print(response.json())
```

Comment:
Instead of having the user uncomment this code, could we put the code inside an `if __name__ == "__main__":` block?